packages/components/src/button/stories/index.js

```diff
@@ -23,142 +18,62 @@ import Button from '../';
 export default {
 	title: 'Components/Button',
 	component: Button,
-	parameters: {
-		knobs: { disable: false },
+	argTypes: {
```

Comment:
Optional for this PR: to make the Storybook examples even more informative, we could add more info for each prop (i.e. `description`, default values...). Both descriptions and default values could be copy-pasted from the README.

You can look at how `FontSizePicker` does this in its stories here (the relevant bits are the `argTypes` and `parameters` objects): 

https://github.com/WordPress/gutenberg/blob/c64569089335e76da642fb39b153e30d069d2d00/packages/components/src/font-size-picker/stories/index.js#L14-L37